home/utils/UrlUtils.ts

```diff
@@ -1,6 +1,8 @@
 import url from 'url';
 
-const HTTPS_HOSTS = ['exp.host', 'exponentjs.com', 'getexponent.com'];
+import Config from '../api/Config';
+
+const HTTPS_HOSTS = [Config.api.host, 'exponentjs.com', 'getexponent.com'];
```

Comment:
I'd leave exp.host in the list and just have it be duplicated since we likely will change Config.api.host in the future and want old exp.host URLs to be treated the same way.

Reply:
That seems like a good idea. I've re-added `exp.host` 👍